account_analytic_distribution/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+Account Analytic Distribution
+=============================
+
+This module is a simpler alternative to official 'account_analytic_plans'
```

Comment:
Change this text, as now in version 9.0, there's no account_analytic_plans module, and also explains the purpose (as if you don't know account_analytic_plans).